Makefile

```diff
@@ -41,4 +38,27 @@ lint:
 install:
 	for d in $(SUBDIRS); do $(MAKE) -C $$d install; done
 
-.PHONY: all $(SUBDIRS) clean proto deps lint install
+sandbox-test-cri-build: sandbox-test-cri-build-stamp
+
+sandbox-test-cri-build-stamp:
+	docker build -f sandbox/cri/Dockerfile -t "localhost/sandbox-test-cri" .
+	@touch sandbox-test-cri-build-stamp
+
+sandbox-test-cri-run: sandbox-test-cri-build
+	test "$(shell id --user)" -eq 0
+	docker run \
+		--init \
+		--rm \
+		--privileged \
```

Comment:
Do we really, really need `--privileged`? Can we get away with some added capabilities instead?

Reply:
Can you elaborate on what you mean by this? We only need this for mounting.

Reply:
That's exactly my point. We do not need all the access that comes with `--privileged` just to call mount.

Reply:
Ah, I tried to remove `--privileged` and add sudo to the command, but that fails. /=

Reply:
As expected. `sudo` doesn't mean anything, especially considering that you're already running as root inside the container. The container itself is constrained. There are several ways to remove some or all of the constraints. `--privileged` removes them all, but that's not usually necessary. In this case, you definitely don't need to run with full privileges. Try adding `CAP_SYS_ADMIN` via docker's `--cap-add` flag.

Reply:
Hm, that still doesn't work for me. Still get the good ol' `mount failed: Unknown error -1`.